Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.2-alpine3.12
+FROM ruby:2.7.2-alpine3.13
```

Comment:
Is it worth unfixing this back to `FROM ruby:2.7.2` as well?

Reply:
Good call - I've removed the alpine version number